app/views/layouts/_user_input_filter.html.haml

```diff
@@ -76,7 +76,7 @@
                            :alt   => t = _("No input to apply"),
                            :title => t)
             %span#quick_search_buttons_on{:style => "display:none;"}
-              = link_to(t,
+              = link_to(t = _("Apply"),
```

Comment:
@romanblanco i think you can eliminate
```
t = 
``` 
it is being set on https://github.com/romanblanco/manageiq-ui-classic/blob/b878f36a072169bf3b2dffa53b5e91ec75715842/app/views/layouts/_user_input_filter.html.haml#L83